scripts/grafana/install

```diff
@@ -2,21 +2,37 @@
 
 set -e
 
-GRAFANA_VERSION=5.1.4
+GRAFANA_VERSION=6.0.0
 
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 # Install Grafana dependencies
 sudo apt-get install libfontconfig1 fontconfig-config fonts-dejavu-core ttf-bitstream-vera fonts-liberation sqlite3 -y
 
-# Download and install unofficial build of Grafana for ARM
-URL="https://bintray.com/fg2it/deb/download_file?file_path=main%2Fg%2Fgrafana_${GRAFANA_VERSION}_armhf.deb"
-if uname -m | grep -Eq ^armv6; then
-    # Raspberry Pi 1
-    URL="https://bintray.com/fg2it/deb-rpi-1b/download_file?file_path=main%2Fg%2Fgrafana_${GRAFANA_VERSION}_armhf.deb"
-elif uname -m | grep -Eq ^aarch64; then
-    URL="https://bintray.com/fg2it/deb-arm64/download_file?file_path=main%2Fg%2Fgrafana_${GRAFANA_VERSION}_arm64.deb"
-fi
+ARCH="$(uname -m)" 
+case "$ARCH" in
+  x86_64)
+    URL="https://dl.grafana.com/oss/release/grafana_${GRAFANA_VERSION}_amd64.deb"
+  ;;
+  i386 | i586 | i686 )
+   echo "Cannot install grafana on i386"
+   exit 0
+  ;;
+  armv7l)
+    URL="https://dl.grafana.com/oss/release/grafana_${GRAFANA_VERSION}_armel.deb";
+  ;;
+  armv6l)
+    # Use old grafana - no new one available
+    URL="https://bintray.com/fg2it/deb-rpi-1b/download_file?file_path=main%2Fg%2Fgrafana_5.1.4_armhf.deb"
+  ;;
+  aarch64)
+    URL="https://dl.grafana.com/oss/release/grafana_${GRAFANA_VERSION}_arm64.deb";
+  ;;
+  *)
+    echo "Unknown Arch"
+    exit 1
```

Comment:
You use `exit 0` up above for when you can't download because of i386. Does the code make any difference?

Reply:
Difference is as folows:

0 will exit the script but continue executing install  (we KNOW i386 is not available so  just continue)
1 will stop execution the install script (Something went wrong if we dont know what ARCH it is!)